datanode/sqlsubscribers/party.go

```diff
@@ -30,8 +31,14 @@ type PartyEvent interface {
 	Party() types.Party
 }
 
+type PartyProfileUpdatedEvent interface {
+	events.Event
+	PartyProfileUpdated() eventspb.PartyProfileUpdated
```

Comment:
Core method have the following signature
```suggestion
	PartyProfileUpdated() *eventspb.PartyProfileUpdated
```